src/pages/user/newUser.tsx

```diff
@@ -1,5 +1,215 @@
-const NewUser = (): JSX.Element => {
-  return <div>userLogin</div>;
-};
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
 
-export default NewUser;
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { useForm } from "react-hook-form";
+
+export default function NewUser() {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm();
+
+  const submitLogin = (data: any) => {
+    console.log("data", data);
+    reset({
+      name: "",
+      email: "",
+      password: "",
+      language: "",
+      phone: "",
+      address: "",
+      sex: "",
+      dob: "",
+      avatar: "",
+      roleId: "",
+      username: "",
+    });
+  };
+
+  return (
+    <div className="flex mt-16 justify-center items-center">
+      <Card className="w-[450px]">
+        <form onSubmit={handleSubmit(submitLogin)}>
+          <CardHeader>
+            <CardTitle>Sign Up</CardTitle>
+            <CardDescription>New User registeration</CardDescription>
+          </CardHeader>
+          <CardContent>
+            <div className="grid w-full items-center gap-4">
+              <div className="flex flex-col space-y-1.5">
+                <Label htmlFor="name">Name</Label>
+                <Input
+                  id="name"
+                  placeholder="Name"
+                  {...register("name", { required: true })}
+                />
+                {errors.name && errors.name.type === "required" && (
+                  <p className="text-xs text-rose-500 text-left">
+                    Name is required
+                  </p>
+                )}
+              </div>
+              <div className="flex flex-col space-y-1.5">
+                <Label htmlFor="username">User Name</Label>
+                <Input
+                  id="username"
+                  placeholder="User Name"
+                  {...register("username", { required: true })}
+                />
+                {errors.username && errors.username.type === "required" && (
+                  <p className="text-xs text-rose-500 text-left">
+                    User Name is required
+                  </p>
+                )}
+              </div>
```

Comment:
just add confirm password logic too

Reply:
Added validation and confirm password logic using yup
Thanks :)